react/src/components/LegacyFolderExplorer.tsx

```diff
@@ -191,7 +193,7 @@ const LegacyFolderExplorer: React.FC<LegacyFolderExplorerProps> = ({
             {vfolder_node?.user === currentUser.uuid && (
               <BAISelect
                 tooltip={t('data.folders.MountPermission')}
-                defaultValue={vfolder_node?.permission || vfolder?.permission}
+                defaultValue={permission === 'wd' ? 'rw' : permission}
```

Comment:
How about logging warning message when `wd` value is received, if the wd permission is not allowed here?

Also, I think rather than changing the permission to `rw`, it seems better to keep the permission as is and change the display format of `wd` to `ReadWrite`.

Reply:
I'll add a warning message. And if I keep the permission and change the display format of `wd`, it will be shown as below. So I'll leave as it is.
 ![alt text](image.png)